test/lit.cfg

```diff
@@ -129,6 +132,43 @@ if config.test_exec_root is None:
 
 ###
 
+class SwiftTest(lit.formats.ShTest):
+    def __init__(self, coverage_mode=None, execute_external=True):
```

Comment:
I'm not sure I understand--`SwiftTest` is only ever initialized once below, and no `coverage_mode` is specified. Is the `coverage_mode` parameter ever used?